app/services/idv/vendor.rb

```diff
@@ -0,0 +1,24 @@
+module Idv
+  class Vendor
+    # Proofer gem will attempt to autoload based on the initializer name
+    # so make sure our load path includes whichever vendors we have defined.
+    if Figaro.env.proofing_vendors
+      Figaro.env.proofing_vendors.split(/\W+/).each do |vendor|
+        vendor_path = "#{Rails.root}/vendor/#{vendor}/lib"
+        $LOAD_PATH.unshift vendor_path
+      end
+    end
+
+    def pick
+      if Rails.env.test?
+        :mock
```

Comment:
Is this necessary? The test environment already has `proofing_vendors` defined in application.yml, so `available.sample` should work for all envs, no?